components/admin_console/member_list_group/member_list_group.scss

```diff
@@ -0,0 +1,18 @@
+.MemberListGroup {
+    .DataGrid {
+        display: flex;
+        flex-direction: column;
+        height: calc(90vh - 120px);
+        width: 100%;
+        padding: 0;
+
+        .DataGrid_rows {
+            flex-grow: 500;
+            height: 1px;
+            overflow-y: scroll;
+        }
+        .DataGrid_empty {
```

Comment:
If this is a "modifier" this should be separated by two underscores (ie `.DataGrid__empty`), [per the apps team's naming convention](https://medium.com/trabe/a-comprehensive-guide-to-using-bem-with-react-14d00c199e0d).